templates/zerver/realm_creation_disabled.html

```diff
@@ -0,0 +1,13 @@
+{% extends "zerver/portico.html" %}
+
+{% block for_you %}{{ _('got a bit lost there.') }}{% endblock %}
+
+{% block portico_content %}
+
+<br/>
+<p class="lead">{{ _('Realm creation disabled') }}</p>
```

Comment:
Yeah, let's call it "organization", since that's what we use everywhere else in the UI (sorry for the confusion here)